packages/react-ui-core/src/Gmap/GmapSpinner.js

```diff
@@ -11,6 +11,7 @@ export default class GmapSpinner extends PureComponent {
     theme: PropTypes.object,
     className: PropTypes.string,
     closeDelay: PropTypes.number,
+    color: PropTypes.string,
```

Comment:
que esta?

Reply:
It was missing from the prop declaration and complained cause of this line: https://github.com/rentpath/react-ui/pull/425/files/ee913a21d8acc8dd276087c0eef5dffde40271b1#diff-d52d99f3141a80bfb67aef7d1694b4a0R21